extern/authorizeIPN.php

```diff
@@ -38,6 +38,7 @@
 
 require_once '../civicrm.config.php';
 $config = CRM_Core_Config::singleton();
+CRM_Utils_Hook::alterIPNData($_REQUEST);
```

Comment:
I don't think we should add hooks into these deprecated paths - there is some risk of them not working from previous nightmares with these bits of code so I'd rather not touch (also people should stop using them)